src/cache/cache_las.c

```diff
@@ -624,6 +624,13 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
 
     /* Enter each update in the boundary's list into the lookaside store. */
     for (i = 0, list = multi->supd; i < multi->supd_entries; ++i, ++list) {
+        /* If no onpage_upd is selected, we don't need to insert anything to lookaside */
```

Comment:
This `onpage_upd` struct has a `WT_UPDATE` pointer in the case where we've selected from the update list and some supplementary information like timestamps, etc on the `onpage_upd` directly to account for when we select a lookaside update. Since you only ever select from the update list now, this struct should be cleaned up.

Reply:
387e650